mne/viz/utils.py

```diff
@@ -321,11 +323,18 @@ def _get_help_text(params):
     text.append(u'\u2191 : \n')  # up arrow
     text.append(u'- : \n')
     text.append(u'+ or = : \n')
-    text.append(u'Home : \n')
-    text.append(u'End : \n')
-    if 'fig_selection' not in params:
-        text.append(u'Page down : \n')
-        text.append(u'Page up : \n')
+    if is_mac:
+        text.append(u'fn + \u2190 : \n')
+        text.append(u'fn + \u2192 : \n')
```

Comment:
Probably cleaner just to put in the chars (e.g. ←) directly. You might need to add `# -*- coding: utf-8 -*-` to the top of the file